docs/api/tasks-get-tasks-by-user_id_david.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+---
+
+# `task` resource - Get Task by User ID
+
+Base endpoint:
+
+```shell
+{base_url}/tasks?user_id={user_id}
+```
+
+Retrieves all tasks associated with a specific user based on the provided `user_id`.
+
+To use this endpoint, the user must be added to the service first. Learn more about the [user resource](user.md).
+
+## Request
+
+### Parameters
+
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| `user_id` | number | The ID of the user whose tasks should be retrieved |
+
+Example request:
+
+```shell
+curl http://localhost:3000/tasks?user_id=1
+```
+
+## Response
+
+A successful response returns a list of tasks associated with the specified `user_id`. If no tasks are found, an empty array is returned.
+
+### Sample Response
+
+```json
+[
+  {
+    "user_id": 1,
+    "title": "Grocery shopping",
+    "description": "eggs, bacon, gummy bears",
+    "due_date": "2024-02-20T17:00",
+    "warning": "-10",
+    "id": 1
+  },
+  {
+    "user_id": 1,
+    "title": "Piano recital",
+    "description": "Daughter's first concert appearance",
+    "due_date": "2024-04-02T15:00",
+    "warning": "-30",
+    "id": 2
+  }
+]
+```
+
+### Error Response
+
+If the `user_id` is not found or invalid, an error message will be returned.
+
+#### Sample Error Response
+
+```json
+[]
+```
+
+## Resource Properties
```

Comment:
This section is handy, but consider the maintenance impact of content that's duplicated on the resource reference page. A lower-maintenance option is to link to that section of the resource reference page instead of duplicating the content.

A higher-tech, but not always supported, option is to make the property description table a snippet that's imported into each page in which you want to display it. Then you're back to only having one copy top maintain with the advantage of showing that copy in various locations. I think the static site generator used to create the GitHub Pages content can do that, but I don't recall the syntax off the top of my head.